docs/tutorials/getting_started/customize_your_deployment.md

```diff
@@ -2,7 +2,7 @@
 title: Optional - Customize your deployment
 ---
 
-At this point, you have your first, working local deployment of Great Expectations. You’ve also been introduced to the foundational concepts in the library: [Data Contexts](/docs/reference/data_context), [Datasources](/docs/reference/datasources), [Expectations](/docs/reference/expectations/expectations), [Profilers](/docs/reference/profilers), [Data Docs](/docs/reference/data_docs), [Validation](/docs/reference/validation), and [Checkpoints](/docs/reference/checkpoints_and_actions).
+At this point, you have your first, working local deployment of Great Expectations. You’ve also been introduced to the foundational concepts in the library: [Data Contexts](../../reference/data_context.md), [Datasources](../../reference/datasources.md), [Expectations](../../reference/expectations/expectations.md), [Profilers](../../reference/profilers.md), [Data Docs](../../reference/data_docs.md), [Validation](../../docs/reference/validation.md), and [Checkpoints](../../reference/checkpoints_and_actions.md).
```

Comment:
Validation link should be `../../reference/validation.md`

Reply:
I fixed the link.